airflow_client/client/api/__init__.py

```diff
@@ -1,3 +1,30 @@
-# do not import all apis into this module because that uses a lot of memory and stack frames
-# if you need the ability to import all apis from one package, import them with
-# from airflow_client.client.apis import ConfigApi
+# flake8: noqa
+
+# import apis into api package
+from airflow_client.client.api.asset_api import AssetApi
+from airflow_client.client.api.backfill_api import BackfillApi
+from airflow_client.client.api.config_api import ConfigApi
+from airflow_client.client.api.connection_api import ConnectionApi
+from airflow_client.client.api.dag_api import DAGApi
+from airflow_client.client.api.dag_parsing_api import DAGParsingApi
+from airflow_client.client.api.dag_report_api import DagReportApi
+from airflow_client.client.api.dag_run_api import DagRunApi
+from airflow_client.client.api.dag_source_api import DagSourceApi
+from airflow_client.client.api.dag_stats_api import DagStatsApi
+from airflow_client.client.api.dag_version_api import DagVersionApi
+from airflow_client.client.api.dag_warning_api import DagWarningApi
+from airflow_client.client.api.event_log_api import EventLogApi
+from airflow_client.client.api.extra_links_api import ExtraLinksApi
+from airflow_client.client.api.import_error_api import ImportErrorApi
+from airflow_client.client.api.job_api import JobApi
+from airflow_client.client.api.login_api import LoginApi
+from airflow_client.client.api.monitor_api import MonitorApi
+from airflow_client.client.api.plugin_api import PluginApi
+from airflow_client.client.api.pool_api import PoolApi
+from airflow_client.client.api.provider_api import ProviderApi
+from airflow_client.client.api.task_api import TaskApi
+from airflow_client.client.api.task_instance_api import TaskInstanceApi
+from airflow_client.client.api.variable_api import VariableApi
+from airflow_client.client.api.version_api import VersionApi
+from airflow_client.client.api.x_com_api import XComApi
+
```

Comment:
Something I don't understand. This looks correct, but when I take a closer look, I don't see any file corresponding to `from airflow_client.client.api.asset_api import AssetApi`. That's also the case for some other APIs, some of them exist some does not.

This yield an error when I try to run the test_python_client.py script.

Reply:
Fixed, test passes